web-marketplace/src/mocks/handlers.ts

```diff
@@ -0,0 +1 @@
+export const handlers = [];
```

Comment:
here we would add request handlers, similarly to the ones I've set up in the .stories.tsx below